src/main/kotlin/org/rust/lang/core/psi/RustLiteral.kt

```diff
@@ -0,0 +1,8 @@
+package org.rust.lang.core.psi
+
+import com.intellij.psi.PsiElement
+import com.intellij.psi.tree.IElementType
+
+interface RustLiteral : PsiElement {
+    val tokenType: IElementType
+}
```

Comment:
An alternative design would be to empoy a sealed class here (to express in type system that the set of literals is closed)

```
abstract sealed class RustLiteral : PsiElemnt, LeafPsiElement {
   abstract class RustNumberLiteral: RustLiteral
   abstract class RustStringLiteral: RustLiteral
}
```

I don't know if we should go this way.